common/v2/services/Store/StoreProvider.tsx

```diff
@@ -326,7 +326,9 @@ export const StoreProvider: React.FC = ({ children }) => {
       return sortBy(prop('ticker'), uniq);
     },
     assets: (selectedAccounts = state.accounts) =>
-      selectedAccounts.flatMap((account: StoreAccount) => account.assets),
+      selectedAccounts
+        .flatMap((account: StoreAccount) => account.assets)
+        .filter((asset) => !asset.balance.eq(0)),
```

Comment:
This is only hides them from the view.
Wouldn't we want to delete them from storage instead?